frontend/compositions/visualizations/popUps/popUpComp.tsx

```diff
@@ -0,0 +1,39 @@
+import { useCallback, useEffect, useRef, useState } from "react"
+import { getTitleCaseFromCamel } from "../../../helpers"
+import { Coord } from "../utilities/chartTypes"
+import styles from "./popUps.module.css"
+
+export interface PopUpProps {
+  shouldShowPopUp?: boolean
+  headerText?: string
+  bodyText?: string
+  location?: Coord
+}
+
+export default function PopUp({ shouldShowPopUp, headerText, bodyText, location }: PopUpProps) {
+  const classNames = `${styles.popUp} ${shouldShowPopUp ? styles.popUpOpen : styles.popUpClosed}`
+
+  const ariaLabel: string = `Detailed information about ${headerText}`
+  const popUpBodyId: string = `${headerText}Details`
+
+  const style = {
```

Comment:
Could we do this through a css file instead?

Reply:
not that I know of -- the location is passed in as a prop, and transforms overwrite each other so the translate X needs to be stated here, too.